addons/hr_work_entry/models/hr_work_entry.py

```diff
@@ -17,23 +17,24 @@
 class HrWorkEntry(models.Model):
     _name = 'hr.work.entry'
     _description = 'HR Work Entry'
-    _order = 'conflict desc,state,date'
+    _order = 'create_date'
 
-    name = fields.Char(required=True, compute='_compute_name', store=True, readonly=False, precompute=True)
+    name = fields.Char(default=False, store=True, readonly=False)
```

Comment:
Remaining parameters are default values.
```suggestion
    name = fields.Char()
```